helm/coco-kafka-bridge/app-configs/kafka-bridge_publishing_staging_eu.yaml

```diff
@@ -6,7 +6,7 @@ bridges:
 # todo remove all bridges once we are 100% live and methode publishes directly to this staging. 
 ##CMS
 - name: cms-kafka-bridge-pub-staging-eu
```

Comment:
Could you please rename these modified bridges to reflect that they connect now to prod?

Reply:
I have now made these changes.